app/bio/page.tsx

```diff
@@ -1,28 +1,78 @@
 import React from "react";
 import Image from "next/image";
 import biopic from "../public/biopic.jpg";
-import "../globals.css";
+import biopic2 from "../public/biopic2.jpg";
+import { Montserrat } from "next/font/google";
+import "../styles.css";
 import "tailwindcss/tailwind.css";
 
+const montserrat = Montserrat({
+  weight: "400",
+  preload: false,
+});
+
 export default function Bio() {
   return (
     <main>
-      <h2>Bio</h2>
-      <div style={{ display: "flex", flexDirection: "row", gap: "16px" }}>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyItems: "center",
+          gap: "16px",
+        }}
```

Comment:
Inline styles like these are great for quick development and debugging, but should be made into a formal CSS class name that you can reference. Feel free to create a new sheet that's for your styles only, and create them them there using Tailwind's built-ins. Then import it alongside the `styles.css` file and you can access them in your React pages.

Reply:
edited and committed changes